libcontainer/resourcemanager/resource_manager_linux.go

```diff
@@ -0,0 +1,34 @@
+// +build linux
```

Comment:
After a second check, I think current implementation is fine, it's only imported by packages which are wrapped by `linux` build flag, sorry for the noise.